src/utils/webrtc/index.js

```diff
@@ -263,7 +265,7 @@ async function signalingJoinCall(token, flags, silent, recordingConsent) {
 				localMediaModel.setVirtualBackgroundImage(virtualBackgroundUrl)
 			} else if (virtualBackgroundType === VIRTUAL_BACKGROUND.BACKGROUND_TYPE.VIDEO) {
 				localMediaModel.setVirtualBackgroundVideo(virtualBackgroundUrl)
-			} else {
+			} else if (virtualBackgroundType === VIRTUAL_BACKGROUND.BACKGROUND_TYPE.BLUR) {
```

Comment:
I would not add the condition and leave it as a fallback like before; it should not happen that the virtual background is enabled with an invalid type, but in that case it would be better if the background is blurred rather than not having any virtual background at all.

Reply:
Ok, but need to check if that wouldn't force the blur settings on the calls we don't set it, and don't expect therefore